### DISCUSSION TITLE
is there any "SyncList" or "SyncText"  available?

### DISCUSSION THREAD

**Discussion by rozek** [logux/logux#122] [General]
I found "SyncMap", but neither a "SyncList" (similar to Y.js or sync9/antimatter) nor a "SyncText" - however, these would be very important for many applications. Being a complete "Logux" newbie and suffering from the lack of complete examples I prefer to hold back my own attempts to implement them (e.g., similar to what [Seph Gentle](https://josephg.com/blog/crdts-go-brrr/) wrote)

**Comment by ai** [comment]
We don’t have it and do not plan to add in near future.

In cases where we use Logux (converting the standard CRUD webapp) we need mostly Map and List/Text requires too complicated DB structure.

You can see `SyncMap` implementation and implement list. But if you need full CRDT, it is better to use Y.js right now).

**Reply by rozek** [reply]
wow, that was quick!

Nevertheless, that's a pity (although I can understand your point) Personally, I don't know "Logux" well enough to implement it myself...

**Reply by rozek** [reply]
and, yes, Y.js (and its variants) seems to be the best solution right now
